### ISSUE TITLE
Flake8 and stdin

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#48]
In GitLab by @josephbkahn on Oct 6, 2014, 15:29

In 2.2.0 I couldn't pass files through stdin and it was subsequently fixed in 2.2.1.
Since 2.2.2 it's been broken for me and I get this
```bash
Process Process-1:
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "/data/virtualenv/flake8-print/local/lib/python2.7/site-packages/flake8/reporter.py", line 54, in process_main
    self.input_file(filename)
  File "/data/virtualenv/flake8-print/local/lib/python2.7/site-packages/flake8/engine.py", line 79, in input_file
    filename, lines=lines, options=self.options)
  File "/data/virtualenv/flake8-print/local/lib/python2.7/site-packages/pep8.py", line 1230, in __init__
    self.lines = stdin_get_value().splitlines(True)
ValueError: I/O operation on closed file
```
Seems as though it was fixed in bitbucket Issue #155

Looking at the diff between versions I'm not sure what made it not work.

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Oct 9, 2014, 13:00

The fix is currently unreleased, but 2.2.4 should be out today or tomorrow. The problem is with multiprocessing being on by default and that conflicting with being able to properly read from stdin.

**Comment by asottile** [comment]
In GitLab by @josephbkahn on Oct 14, 2014, 19:56

Awesome, Thanks.
